postgres_exporter/linux/queries_backrest.yml

```
@@ -44,10 +47,16 @@ ccp_backrest_last_info:
             description: "Configuration file for this backup"
         - stanza:
             usage: "LABEL"
-            description: "PGBackrest Stanza Name"
+            description: "pgBackRest stanza name"
+        - repo:
+            usage: "LABEL"
+            description: "pgBackRest repository number"
         - backup_type:
             usage: "LABEL"
             description: "Backup type"
+        - backrest_repo_version:
+            usage: "GAUGE"
+            description: "pgBackRest version number"
```

Not a correct description
```suggestion
            description: "pgBackRest version number when this backup was performed"
```
I still don't see any benefit capturing this metric ... confusing in my opinion.